packages/aws-cdk/lib/logging.ts

```diff
@@ -1,43 +1,33 @@
 import * as colors from 'colors/safe';
+import { Writable } from 'stream';
 import * as util from 'util';
 
-// tslint:disable:no-console the whole point of those methods is precisely to output to the console...
```

Comment:
it would be good to provide the rationale for dropping `console.*` and moving to using stdout/stderr and manually adding newlines - maybe in the commit body itself

Reply:
@shivlaks I did exactly that. the commit message says
> console methods use their own set of colors, which CDK then has to override.
Switching to stdout/stderr streams gets rid of this step.